### PR TITLE
Updating the Fine tuning tutorial

### PR DESCRIPTION
**Pull request by TuanaCelik**
In this PR I simplify the fine-tuning tutorial and _only_ focus on the fine tuning task. I propose that we have a _separate_ tutorial for model distillation.

Before merging this PR:
1. We need to make sure that telemetry registers the correct tutorial with the new dataset download step
2. We should also include the separate distillation tutorial within this PR

### CODE REVIEW

**Comment by TuanaCelik** [comment]
@bilgeyucel and @agnieszka-m - I am adding you as reviewers so that we first review the new content. The PR will stay open until we fix 1 and 2 in the PR comment

File: markdowns/02_Finetune_a_model_on_your_data.md [hunk 1]
```diff
@@ -4,55 +4,51 @@ featured: False
...
+completion_time: 15 min
 created_at: 2021-08-12
 ---
     
 
 
+- **Level**: Intermediate
+- **Time to complete**: 15 minutes
+- **Nodes Used**: `FARMReader`
+- **Goal**: After completing this tutorial, you will have learned how to fine-tune a pretrained Reader model with your own data.
```
Comment (agnieszka-m) [inline markdowns/02_Finetune_a_model_on_your_data.md:23]: I think it's "on your own data", not "with"

File: markdowns/02_Finetune_a_model_on_your_data.md [hunk 2]
```diff
@@ -63,102 +59,73 @@ logging.getLogger("haystack").setLevel(logging.INFO)
...
 
 1. **Annotation**: You can use the [annotation tool](https://haystack.deepset.ai/guides/annotation) to label your data, i.e. highlighting answers to your questions in a document. The tool supports structuring your workflow with organizations, projects, and users. The labels can be exported in SQuAD format that is compatible for training with Haystack.
 
-![Snapshot of the annotation tool](https://github.com/deepset-ai/haystack-tutorials/raw/main/tutorials/img/annotation_tool.png)
+![Snapshot of the annotation tool](https://raw.githubusercontent.com/deepset-ai/haystack/main/docs/img/annotation_tool.png)
 
 2. **Feedback**: For production systems, you can collect training data from direct user feedback via Haystack's [REST API interface](https://github.com/deepset-ai/haystack#rest-api). This includes a customizable user feedback API for providing feedback on the answer returned by the API. The API provides a feedback export endpoint to obtain the feedback data for fine-tuning your model further.
 
 
 ## Fine-tune your model
```
Comment (agnieszka-m) [inline markdowns/02_Finetune_a_model_on_your_data.md:73]: Fine-Tune Your Model

File: markdowns/02_Finetune_a_model_on_your_data.md [hunk 3]
```diff
@@ -63,102 +59,73 @@ logging.getLogger("haystack").setLevel(logging.INFO)
...
 Then change the `use_gpu` arguments below to `True`
 
-
-```python
-reader = FARMReader(model_name_or_path="distilbert-base-uncased-distilled-squad", use_gpu=True)
-data_dir = "data/squad20"
-# data_dir = "PATH/TO_YOUR/TRAIN_DATA"
-reader.train(data_dir=data_dir, train_filename="dev-v2.0.json", use_gpu=True, n_epochs=1, save_dir="my_model")
-```
+1. Initialize a `Reader` with the model you would like to finetune
```
Comment (agnieszka-m) [inline markdowns/02_Finetune_a_model_on_your_data.md:83]:
```suggestion
1. Initialize a `Reader` with the model to fine-tune:
```

File: markdowns/02_Finetune_a_model_on_your_data.md [hunk 4]
```diff
@@ -63,102 +59,73 @@ logging.getLogger("haystack").setLevel(logging.INFO)
...
 ```python
-# Saving the model happens automatically at the end of training into the `save_dir` you specified
-# However, you could also save a reader manually again via:
-reader.save(directory="my_model")
+from haystack.nodes import FARMReader
+
+reader = FARMReader(model_name_or_path="distilbert-base-uncased-distilled-squad", use_gpu=True, devices=["mps"])
 ```
 
+2. Get SQuAD style data to be used for training. Below, you can fetch a dataset that we have already prepared.
```
Comment (agnieszka-m) [inline markdowns/02_Finetune_a_model_on_your_data.md:92]: Get SQUAD-style data for training. You can use this dataset we prepared:

Comment (agnieszka-m) [reply markdowns/02_Finetune_a_model_on_your_data.md:92]: Remember our voice :)

File: markdowns/02_Finetune_a_model_on_your_data.md [hunk 5]
```diff
@@ -63,102 +59,73 @@ logging.getLogger("haystack").setLevel(logging.INFO)
...
 
-glove_dir = "data/tutorial2/gloves"
-squad_dir = "data/tutorial2/squad_small"
+reader.train(data_dir=data_dir, train_filename="squad20/dev-v2.0.json", use_gpu=True, n_epochs=1, save_dir="my_model")
+```
 
-# Downloading smaller glove vector file (only for demonstration purposes)
-glove_url = "https://nlp.stanford.edu/data/glove.6B.zip"
-fetch_archive_from_http(url=glove_url, output_dir=glove_dir)
+4. Now initialize a new reader with your fine-tuned model
```
Comment (agnieszka-m) [inline markdowns/02_Finetune_a_model_on_your_data.md:115]:
```suggestion
4. Now initialize a new reader with your fine-tuned model:
```

File: markdowns/02_Finetune_a_model_on_your_data.md [hunk 6]
```diff
@@ -63,102 +59,73 @@ logging.getLogger("haystack").setLevel(logging.INFO)
...
+new_reader = FARMReader(model_name_or_path="my_model")
 ```
 
-In this case, we use a multiplication factor of 2 to keep this example lightweight. Usually you would use a factor like 20 depending on the size of your training data. Augmenting this small dataset with a multiplication factor of 2, should take about 5 to 10 minutes to run on one V100 GPU.
-
-### Running distillation
-Distillation in haystack is done in two steps: First, you run intermediate layer distillation on the augmented dataset to ensure the two models behave similarly. After that, you run the prediction layer distillation on the non-augmented dataset to optimize the model for your specific task.
-
-If you want, you can leave out the intermediate layer distillation step and only run the prediction layer distillation. This way you also do not need to perform data augmentation. However, this will make the model significantly less accurate.
+5. Finaly, try using the `new_reader` that was initialized with your fine-tuned model.
```
Comment (agnieszka-m) [inline markdowns/02_Finetune_a_model_on_your_data.md:122]: This sounds like we're unsure it's going to work. I'd rephrase: Finally, use the `new_reader` that was initialized with...

**Review by agnieszka-m — CHANGES_REQUESTED** [review]

**Comment by bilgeyucel** [comment]
Closing this PR for the sake of updated PR #243
